### ISSUE TITLE
/admin/pages not loading

### ISSUE THREAD

**Issue by tonyofbyteball** [getgrav/grav-plugin-admin#2318]
After upgrading the admin plugin v1.9.12 -> v1.10.36 /admin/pages doesn't load any more

<img width="1248" alt="image" src="https://user-images.githubusercontent.com/20998749/191569264-96bd5229-3f8b-4810-97f5-92441facde7b.png">

The site works fine and all other admin pages seem to work fine as well.

Clearing the cache folder doesn't help.

If I go directly to a URL like /admin/pages/home, it opens fine. The issue seems to be limited to loading the list of pages only.

Any tips what I should check or try?

**Comment by mdestafadilah** [comment]
have you done with clear cache in terminal?

```bash
php bin\grav cache
```

**Comment by tonyofbyteball** [comment]
Just tried this command, it didn't help :(

**Comment by mdestafadilah** [comment]
> Just tried this command, it didn't help :(

clear your browser cache too,, and testing with some browser please.

**Comment by tonyofbyteball** [comment]
Did that, even tried in incognito window, no luck..

**Comment by jgonyea** [comment]
Does renaming your `pages` directory to `pages-old` and recreating a new `pages` directory fix the issue?

**Comment by tonyofbyteball** [comment]
No, it doesn't :( The same infinite loader as on the screenshot in my first message.

**Comment by jgonyea** [comment]
What plugins are installed/ enabled?

**Comment by tonyofbyteball** [comment]
Here is the list:

<img width="684" alt="image" src="https://user-images.githubusercontent.com/20998749/195139335-824019c1-e047-4b8f-8e76-b712824c3e60.png">

**Comment by jgonyea** [comment]
> No, it doesn't :( The same infinite loader as on the screenshot in my first message.

I just installed the latest versions of your plugins, and I can enumerate the pages overview just fine.  Do you have a custom theme enabled?  I'm just on quark right now.

**Comment by tonyofbyteball** [comment]
We do use a custom theme (not sure how it could affect the admin plugin though), the whole project is open source https://github.com/obyteorg/new-obyteorg

**Comment by rhukster** [comment]
just install another theme (like quark) and set that to your active theme, and see if things load.  If they do, then yes your theme (and most likely it's blueprints) are the issue.

**Comment by tonyofbyteball** [comment]
I switched to Quark but the issue didn't go away (I did clear the cache)

<img width="370" alt="image" src="https://user-images.githubusercontent.com/20998749/195188540-e8e7fcff-24a4-4e6a-b13c-311033b9cdcd.png">

**Comment by tonyofbyteball** [comment]
Just in case it matters, I see the following errors in the console when I try to open this page:

<img width="644" alt="image" src="https://user-images.githubusercontent.com/20998749/195189702-911c0f15-5226-4668-8887-02ff4c8d0a25.png">

The second error ($toastElement) is also on all other admin pages.

**Comment by mdestafadilah** [comment]
> Just in case it matters, I see the following errors in the console when I try to open this page:
> 
> <img alt="image" width="644" src="https://user-images.githubusercontent.com/20998749/195189702-911c0f15-5226-4668-8887-02ff4c8d0a25.png">
> 
> The second error ($toastElement) is also on all other admin pages.

are you using VPN for browsing?

**Comment by tonyofbyteball** [comment]
I tried to disable the Problems plugin but the problem didn't go away

<img width="699" alt="image" src="https://user-images.githubusercontent.com/20998749/195977899-5334c2fb-b81b-47f7-93ab-4f3d863cf8fe.png">
